frontstage/controllers/case_controller.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+from flask import current_app as app
+import requests
+from structlog import wrap_logger
```

Comment:
I've noticed a few different approaches to defining import order. None are right/wrong but we should probably settle on and use it (Black would be useful for this)